subprojects/gradle/impact/src/test/kotlin/com/avito/impact/ImpactAnalysisTest.kt

```diff
@@ -178,10 +184,10 @@ class ImpactAnalysisTest {
                 AndroidAppModule("standalone_app"),
                 AndroidAppModule(
                     name = "app",
-                    dependencies = """
-                        implementation project(':feature_a')
-                        implementation project(':feature_b')
-                    """
+                    dependencies = setOf(
+                        project(path = ":feature_a", configuration = IMPLEMENTATION),
```

Comment:
do you need to write specific "IMPLEMENTATION" configuration here? maybe omit it as default one?